Virtual Librarian/Virtual Librarian/MainForm.cs

```diff
@@ -64,5 +79,35 @@ private void textBox1_TextChanged(object sender, EventArgs e)
         {
 
         }
+
+        private void imgCamUser_Click(object sender, EventArgs e)
```

Comment:
not sure if these empty methods are left here intentionaly, do you plan to use them?